plugin/storage/es/spanstore/reader.go

```diff
@@ -333,18 +352,17 @@ func (s *SpanReader) multiRead(ctx context.Context, traceIDs []string, startTime
 				return nil, err
```

Comment:
I noticed that the integration tests failed. Perhaps L298 requires a `traceID.String()`?